app/controllers/application_controller.rb

```diff
@@ -120,7 +120,8 @@ def set_locale
       end
     end
     I18n.locale = locale
```

Comment:
Yep. I just edited this to follow the param-style.
I do not think that we should use cookies for this either.
Will fix.